libraries/chain/controller.cpp

```diff
@@ -2825,6 +2826,29 @@ struct controller_impl {
       }
    } FC_CAPTURE_AND_RETHROW() } /// apply_block
 
+   void create_and_send_vote_msg(const block_header_state& bhs) {
```

Comment:
I don't see where this function is called.

Reply:
This PR was created before code for processing IF received block was ready. The PR provides the plumbing. The function will be called in create_block_state_i after the block is validated (https://github.com/AntelopeIO/leap/issues/2071)